content/Assets/Styles/base/_paragraph.scss

```diff
@@ -9,4 +9,8 @@
 p {
     @include rem(margin-bottom, $spacing-default);
     margin-top: 0;
+
+    &:last-child{
+        margin-bottom:0;
```

Comment:
For consistency, can you include a space after the `:`.

Reply:
Also between `:last-child` and `{`.

Reply:
For reference I'm using this:
https://github.com/prettier/prettier-vscode.git

Which fixes the formatting on save.

Reply:
Interestingly, I have that installed too. I wonder why it's not working, or if there's an option I need to configure...?

Reply:
I actually have `prettier` turned off for CSS myself. However, I'm wondering if we're missing some settings within the `.prettierrc` file thus your editors are implying their own configurations.